tests/contrib/pytest/test_pytest.py

```diff
@@ -123,18 +144,13 @@ def item_param():
                 return 42
 
             @pytest.mark.parametrize(
-                'item',
-                [
-                    1,
-                    2,
-                    3,
-                    4,
-                    pytest.param(A("test_name", "value"), marks=pytest.mark.skip),
-                    pytest.param(A("test_name", A("inner_name", "value")), marks=pytest.mark.skip),
-                    pytest.param({"a": A("test_name", "value"), "b": [1, 2, 3]}, marks=pytest.mark.skip),
-                    pytest.param([1, 2, 3], marks=pytest.mark.skip),
-                    pytest.param(item_param, marks=pytest.mark.skip)
-                ]
+            'item',
+            [
+                pytest.param(A("test_name", "value"), marks=pytest.mark.skip),
+                pytest.param(A("test_name", A("inner_name", "value")), marks=pytest.mark.skip),
+                pytest.param(item_param, marks=pytest.mark.skip),
+                pytest.param({"a": A("test_name", "value"), "b": [1, 2, 3]}, marks=pytest.mark.skip),
```

Comment:
Does one of these cases have a circular reference to reproduce the issue?

Reply:
Not yet, will add one though 😄